src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateCreationSettings.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace WcfTestBridgeCommon
+{
+    [Serializable]
+    public class CertificateCreationSettings
+    {
+        public CertificateCreationSettings()
+        {
+            Subjects = new string[] { string.Empty };
+            IsValidCert = true;
+        }
+        public string [] Subjects { get; set; }
+        public DateTime ValidityNotBefore { get; set; }
+        public DateTime ValidityNotAfter { get; set; }
+        public bool IsValidCert { get; set; }
+    }
+}
```

Comment:
It would be nice to use FriendName to indicate the kind of invalid cert. However, I haven't got time figure out how to do it. Don't want to block this checkin.